resources/metadata/credentialdiggerScan.yaml

```diff
@@ -121,4 +121,4 @@ spec:
           - filePattern: "**/report*.csv"
             type: credentialdigger-report
   containers:
-    - image: saposs/credentialdigger:4.14.0
```

Comment:
credentialdiggerScan is maintained by another team. Would be better to let them decide the versions of the docker images